sqs_workers/sqs_env.py

```diff
@@ -60,10 +61,12 @@ class SQSEnv:
 
     def __attrs_post_init__(self):
         self.context = self.context_maker()
+        # https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html
+        retry_config = Config(retries={"max_attempts": 3, "mode": "standard"})
```

Comment:
What are your thoughts on having `retry_max_attempts`  and `retry_mode` as optional attributes of the class, that default to the choices you made here?